di_website/templates/events/event_page.html

```diff
@@ -0,0 +1,103 @@
+{% extends "layouts/default.html" %}
+
+{% load navigation_tags static wagtailcore_tags wagtailimages_tags card_tags %}
+{% get_site_root as site_root %}
+
+{% block body %}
+{% block hero %}
+  {% include 'includes/heroes/hero-post.html' with tags=page.topics.all %}
+{% endblock hero %}
+
+<div class="row row--narrow">
+	<div class="l-sidebar">
+		<div class="l-sidebar__body">
+			
+    <section class="section">
+        <div class="highlight">
+            <ul class="highlight__list">
+                <li><i role="presentation" aria-hidden="true" class="ico ico--14 ico-calendar-slate"></i> 10 December - 12 December 2019</li>
```

Comment:
hmm... uhm... ?? ... You copy-pasted content from the patterns library surge site ... why? Now every event page looks like this, with Loren ipsum stuff in it.

Reply:
This was filler code to test the event listing page. Events page still needs work.